src/components/common/Card/StyledCard.js

```diff
@@ -0,0 +1,81 @@
+import styled, { css } from "styled-components";
+
+export const Container = styled.div`
+  margin: 1.5%;
+  width: ${({ size }) => (size == "large" ? "90%" : "22%")};
+  border-radius: 2px;
+  /* box-shadow: 0 3px 6px rgba(0, 0, 0, 0.16), 0 3px 6px rgba(0, 0, 0, 0.23); */
+  display: flex;
+  flex-direction: column;
+  @media (orientation: portrait) {
+    width: 90%;
+    margin-bottom: 5%;
+  }
+`;
+
+export const Content = styled.div`
+  width: 100%;
+  display: flex;
+  flex-direction: ${({ flexDirection }) =>
+    flexDirection ? flexDirection : "column"};
+  ${({ size }) =>
+    size == "large" &&
+    css`
+      flex-direction: row;
+      justify-content: space-between;
+    `}
+  && {
+    flex-direction: ${({ flexDirection }) =>
+      flexDirection ? flexDirection : "column"};
+  }
+  align-items: center;
+  text-align: ${({ textAlignment }) =>
+    textAlignment ? textAlignment : "left"};
+`;
```

Comment:
Something here is a little off when ran on a local machine. On the home page, large card should have a flex direction of row, but it turns to flex direction of column. I am assuming it has something to do with line 27, but I am unsure. When you have time, run this on your local machine and take a look at the large card that says "Development Labs."